user_tools/pyproject.toml

```diff
@@ -8,15 +8,23 @@ authors = [
   { name="Raza Jafri", email="raza.jafri@gmail.com" },
   { name="Ahmed Hussein", email="a@ahussein.me" },
 ]
-description = "A simple wrapper process around gcloud to process the event logs from local or GCS"
+description = "A simple wrapper process around cloud service providers to run RAPIDS Accelerator for Apache Spark."
 readme = "README.md"
 requires-python = ">=3.8"
 classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: Apache Software License",
     "Operating System :: OS Independent",
 ]
-dependencies = ["fire==0.4.0", "pandas==1.4.3", "pyYAML==6.0", "tabulate==0.8.10", "importlib-resources"]
+dependencies = [
+    "fire==0.4.0",
+    "pandas==1.4.3",
+    "pyYAML==6.0",
+    "tabulate==0.8.10",
+    "importlib-resources==5.10.2",
+    "requests==2.28.2",
+    "wget==3.2"
+]
 dynamic=["entry-points", "version"]
 [tool.setuptools.dynamic]
-version = {attr = "spark_rapids_dataproc_tools.__version__"}
+version = {attr = "pyrapids.__version__"}
```

Comment:
naming convention: should we use pysparkrapids instead of pyrapids to not conflict with other RAPIDS teams and their python code/libraries?

Reply:
Changed to `spark_rapids_pytools`